src/screens/app/App.jsx

```diff
@@ -3,6 +3,7 @@ import './App.css';
 import Routes from '../../routes/routes';
 import setLanguage from '../../trans/trans';
 import Xhr from '../../Xhr';
+import Snackbar from '../../components/snackbar';
```

Comment:
You have here a word case error, the folder is Snackbar, the folder is called Snackbar with a capital S